packages/shared/components/molecules/WalletList.svelte

```diff
@@ -0,0 +1,88 @@
+<script lang="typescript">
+    import { onMount } from 'svelte'
+    import { WalletTile } from 'shared/components'
+    import { getAccounts } from '../../lib/core/profile-manager'
+    import { StardustAccount } from '../../lib/typings/account'
+    import { AccountMeta } from '@iota/wallet/out/types'
+
+    let wallets: StardustAccount[] = [
+        <StardustAccount>{
```

Comment:
This is just some hard-coded data that will be removed once the app logic is hooked up (this will just be an empty array `[]`).